src/client/containers/DeleteBoardModal/DeleteBoardModal.container.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DeleteBoardModal.style.css';
+import CloseButton from '../../components/CloseButton/CloseButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+
+const DeleteBoardModal = ({ boardInfo, setModalVisibility }) => {
+  // const { userId } = useParams();
+
+  const userId = 2;
+  const boardId = 1;
+  const deleteBoard = async (boardId) => {
+    const API_URL = `/api/users/${userId}/boards/${boardId}}`;
```

Comment:
You have an extra '}' in the api url

Reply:
Yes I am testing the code on another branch and just pasting it here. so not realising about these error